src/ip_address.h

```diff
@@ -38,7 +38,7 @@
 
 #include <unordered_map>
 
-#define MAX_UDS_NAME 108
+#define MAX_UDS_NAME 107
```

Comment:
This conflicts with `struct sockaddr_un` definition in `/usr/include/linux/un.h`. For example, on Ubuntu 20.04 this structure are defined as
```C
#define UNIX_PATH_MAX   108

struct sockaddr_un {
    __kernel_sa_family_t sun_family; /* AF_UNIX */
    char sun_path[UNIX_PATH_MAX];   /* pathname */
};
```

Reply:
can we do something as
#define MAX_UDS_NAME (sizeof(sockaddr_un) - sizeof(__kernel_sa_family_t ))
with Windows adaption

Reply:
Removed the MAX_UDS_NAME completely. No real need for it.